packages/web-components/src/components/cta-section/cta-section-item.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, css, customElement } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings.js';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
+import styles from './cta-section.scss';
+import DDSContentItem from '../content-item/content-item';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * List item version of Content Item.
+ *
+ * @element dds-cta-section-item
+ */
+@customElement(`${ddsPrefix}-cta-section-item`)
+class DDSCTASectionItem extends StableSelectorMixin(DDSContentItem) {
+  render() {
+    return html`
+      <div class="${prefix}--content-item">
+        <slot name="heading"></slot>
+        <slot></slot>
+        <slot name="cta"></slot>
+      </div>
```

Comment:
`<cta-section-item>` itself is a DOM element and thus rendering `<div class="bx--content-item">` will be a duplicate. We can use a tag selector to style `<cta-section-item>`: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.12.0/packages/web-components/docs/coding-conventions.md#custom-element-itself-as-an-eleement

```suggestion
      <slot name="heading"></slot>
      <slot></slot>
      <slot name="cta"></slot>
```